processor/src/app.js

```diff
@@ -2,15 +2,15 @@ const express = require('express')
 
 const { createProjectsAPI } = require('./projectAPI')
 const { createRemoteAPI } = require('./remoteAPI')
-const { ProcessingManager } = require('./processingManager')
+const { isRepoReadyForProcessing } = require('./isReadyForProcessing')
```

Comment:
I generally try and follow this rule: `is..` is a boolean variable, never a function. So I would suggeest: `checkIsRepoReady` and call the file `checkIsRepoReady`.